sql/database.sql

```diff
@@ -6315,16 +6323,18 @@ CREATE TABLE `lists` (
   `injury_type` varchar(31) NOT NULL DEFAULT '',
   `injury_grade` varchar(31) NOT NULL DEFAULT '',
   `reaction` varchar(255) NOT NULL DEFAULT '',
+  `verification` VARCHAR(36) NOT NULL DEFAULT '',
   `external_allergyid` INT(11) DEFAULT NULL,
   `erx_source` ENUM('0','1') DEFAULT '0' NOT NULL  COMMENT '0-OpenEMR 1-External',
   `erx_uploaded` ENUM('0','1') DEFAULT '0' NOT NULL  COMMENT '0-Pending NewCrop upload 1-Uploaded TO NewCrop',
   `modifydate` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
   `severity_al` VARCHAR( 50 ) DEFAULT NULL,
   `external_id` VARCHAR(20) DEFAULT NULL,
-  `list_option_id` VARCHAR(100) DEFAULT NULL COMMENT 'Reference to list_options table',
+  `list_option_id` VARCHAR(100) DEFAULT NULL COMMENT 'Reference to list_options option_id = allergyintolerance-verification',
```

Comment:
I don't think list_option_id is related to allergyintolerance-verification .

Reply:
i think this comment is meant for the verification column above.

Reply:
Thanks :)
Also for other do we need to update Comments in Upgrades or Database.sql is fine?

Reply:
lets keep the old comment, though :)